pkg-manager/plugin-commands-installation/src/update/getUpdateChoices.ts

```diff
@@ -8,6 +8,7 @@ import isEmpty from 'ramda/src/isEmpty'
 export interface ChoiceRow {
   name: string
   value: string
+  path?: string
```

Comment:
where is this coming from? Enquirer already has this field?

Reply:
yes, i can use this filed to judge if the choice from 'packageManager':

![image](https://github.com/pnpm/pnpm/assets/32598811/ffe49121-4f0c-42d2-a00e-dad7a235e9b3)